man/crystal.1

```diff
@@ -0,0 +1,230 @@
+.\"Crystal Programming Language
+.Dd March 7, 2016
+.Dt CRYSTAL(1) "" "Crystal Compiler Command Line Reference Guide"
+.\".Dt CRYSTAL 1
+.Os UNIX
+.Sh NAME
+.Nm crystal - Compiler for the Crystal language.
+.Sh SYNOPSYS
+.Nm
+command
+.Op switches
+programfile
+--
+.Op arguments
+.Sh DESCRIPTION
+Crystal is a statically type-checked programming language. It was created with the beauty of ruby and the performance of pure C in mind.
+.Sh USAGE
+You can compile and run a program by invoking the compiler with a single filename:
+.Bd -offset indent-two
+.Nm
+some_program.cr
+.Ed
+
+Crystal files usually end with the .cr extension.
+
+Alternatively you can use the run command:
+.Bd -offset indent-two
+.Nm
+run
+some_program.cr
+.Ed
+
+To create an executable use the build command:
+.Bd -offset indent-two
+.Nm
+build
+some_program.cr
+.Ed
+
+This will create an executable named "some_program".
+
+Note that by default the generated executables are not fully optimized.
+To turn optimizations on, use the --release flag:
+.Bd -offset indent-two
+.Nm
+--release
+build
+some_program.cr
+.Ed
+
+Make sure to always use --release for production-ready executables and when performing benchmarks.
+
+The reason for this is that performance without full optimizations is still pretty good and provides fast compile times, so you can use the crystal command almost as if it were an interpreter.
```

Comment:
"The optimizations are not turned by default because the compile times are much faster without them and the performance of the program is still pretty good without them, so it allows to use the crystal command almost to be used as if it was an interpreter."